test/helpers/components/operator/installation.go

```diff
@@ -38,20 +38,25 @@ func InstallViaHelm(releaseTag string, withCsi bool, namespace string) features.
 func UninstallViaMake(withCSI bool) features.Func {
 	return func(ctx context.Context, t *testing.T, envConfig *envconf.Config) context.Context {
 		rootDir := project.RootDir()
-		execMakeCommand(t, rootDir, "undeploy/helm", fmt.Sprintf("ENABLE_CSI=%t", withCSI))
+		err := execMakeCommand(t, rootDir, "install", fmt.Sprintf("ENABLE_CSI=%t", withCSI))
```

Comment:
Am i missing something or why do we call `make install` here?

Reply:
lol, bad copy paster good catch!

Reply:
fixed in 59591ea3